devel/githubContrib.go

```diff
@@ -0,0 +1,253 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// go run githubContrib.go to update Contributions.txt
+
+// This script goes from org -> repos (skipping forks) -> contributors -> user
+// -> guess/normalize the company and count contribs
+
```

Comment:
should you also add a bazel BUILD file with a `go_binary` here?  that would allow `bazel run :githubContrib`.